### PR TITLE
add new activities, fixes #3

### LINKED ISSUE #3: supporting other activities
I would like to know what entails supporting other activities (most of which done have gps data (swim, pilates), and some do (mountain bike)). 

I created a small script to see how many of my activites would be converted:

```
#!/bin/bash
MYTMP=$(mktemp)
# echo "Temp file: $MYTMP"

for FILEJSON in ../Sport-sessions/*.json; do
    type=`cat $FILEJSON | jq -c '.sport_type_id'`
    echo $type >> $MYTMP
    # echo $type 
done

echo "Count and activity ids:"
sort $MYTMP | uniq -c | sort -bgr
```

outputs something like:
> Count and activity ids:
>     537 "18"
>     319 "34"
>     262 "31"
>     198 "23"
>     106 "4"
>      75 "1"
>      59 "27"
>      47 "3"
>      34 "5"
>      26 "7"
>      20 "19"
>       5 "15"

so I would like to get most of them covered. I am not proficient in js.
thanks!

### PR DESCRIPTION
**Pull request by jmlucjav**
there is a typo, please fix it, should be:

tennis: '23',

I was able to identify all my activities barring 27. Runtastic removed the web version and the app does not let me go back enough to see a 27 activity to see what it was....

### CODE REVIEW

**Review by glennreyes — COMMENTED** [review]
Awesome, thanks! Could you also please add the activities to the list in the readme file?

**Comment by mipapo** [comment]
Pilates and Tennis have the same number? @jmlucjav

File: index.js [hunk 1]
```diff
@@ -15,6 +15,16 @@ const SPORT_TYPES = {
...
   weight_training: '34',
+  swim: '18',
+  //there is no pilates,tennis, mountain_bike
+  pilates: '31',
+  //no tennis either (23)
+  tennis: '31',
+  mountain_bike: '4',
+  other: '5',
+  hike: '7',
+  cycling_ergometer: '15',
```
Comment (glennreyes) [inline index.js:26]: Can you check if the numbers are right? This seem to be a duplicate with `indoor_ride`.

File: index.js [hunk 2]
```diff
@@ -15,6 +15,16 @@ const SPORT_TYPES = {
   run: '1',
   walk: '19',
   weight_training: '34',
+  swim: '18',
+  //there is no pilates,tennis, mountain_bike
+  pilates: '31',
+  //no tennis either (23)
+  tennis: '31',
```
Comment (glennreyes) [inline index.js:22]: Duplicate as mentioned here https://github.com/glennreyes/runtastic-gpx/pull/4#issuecomment-520431995

**Review by glennreyes — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (jmlucjav) [reply index.js:26]: I mentioned in PR comment:

> there is a typo, please fix it, should be:
> 
> tennis: '23',

**Comment by glennreyes** [comment]
Closing this, since it's now superseded by #5

**Pull request closed without merging**